src/components/QuestionSetItem.js

```diff
@@ -59,36 +56,33 @@ ItemControls.propTypes = {
   onEditClicked: PropTypes.func
 }
 
-const QuestionSetItem = ({ questionSet, onEditClicked, theme }) => {
+const QuestionSetItem = ({ title, children, onEditClicked, theme }) => {
   const headerStyle = {
     borderColor: theme.palette.primary.main
   }
 
-  const { name, questions } = questionSet
-
   return (
     <div className="question-set-item">
       <Paper
         elevation={2}
         className="question-set-item__header"
         style={headerStyle}
       >
-        <h3 className="question-set-item__title">{name}</h3>
+        <h3 className="question-set-item__title">{title}</h3>
         <div className="question-set-item__controls">
           <ItemControls onEditClicked={onEditClicked} />
         </div>
       </Paper>
       <div className="question-set-item__content">
-        <Paper elevation={1}>
-          <QuestionsTable questions={questions} />
-        </Paper>
+        <Paper elevation={1}>{children}</Paper>
```

Comment:
Decided to inject the table as `children` instead of making `QuestionSetItem` render them because I wanted to re-use this component both registration and peer review admin page.